anta/tests/bfd.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""
+BFD test functions
+"""
+# Mypy does not understand AntaTest.Input typing
+# mypy: disable-error-code=attr-defined
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+from anta.models import AntaCommand, AntaTest
+
+
+class VerifyBFDPeersHealth(AntaTest):
+    """
+    Verifies there is no IPv4 BFD peer in the down state, remote disc is not zero and last down should be above the threshold for all VRF.
```

Comment:
What does `remote disc` mean?

Reply:
is it ok to mention `discriminator value of the remote system`

Reply:
Yes perfect